charts/mlflow/templates/deployment.yaml

```diff
@@ -66,22 +66,26 @@ spec:
             - name: http
               containerPort: 5000
               protocol: TCP
+          {{- if .Values.livenessProbe.enabled }}
           livenessProbe:
             httpGet:
-            {{- if .Values.extraArgs.staticPrefix }}
-              path: {{ .Values.extraArgs.staticPrefix }}
-            {{- else }}
-              path: /
-            {{- end }}
+              path: {{ .Values.livenessProbe.healthCheckPath default "/" }}
               port: 5000
+            initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds | default 30 }}
```

Comment:
I think we can make the liveness probe configurable. But default value must be Kubernetes' default value. You can find them [from here](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes).